move-mutator/src/coverage.rs

```diff
@@ -0,0 +1,207 @@
+use crate::compiler::compile_package;
+use anyhow::Error;
+use codespan::Span;
+use move_command_line_common::files::FileHash;
+use move_compiler::compiled_unit::{CompiledUnit, NamedCompiledModule};
+use move_coverage::{
+    coverage_map::CoverageMap,
+    source_coverage::{FunctionSourceCoverage, SourceCoverageBuilder},
+};
+use move_model::model::Loc;
+use move_package::BuildConfig;
+use std::{
+    collections::{BTreeMap, HashMap},
+    fs,
+    path::{Path, PathBuf},
+};
+
+/// Associated function string.
+type AssociatedFuncStr = String;
+
+/// Contains all uncovered spans in the project.
+#[derive(Debug, Default)]
+pub(crate) struct Coverage {
+    /// List of all uncovered spans for all functions for all modules.
+    all_uncovered_spans: BTreeMap<AssociatedFuncStr, UncoveredSpans>,
```

Comment:
What is "associated function string"? Is it the function name? Why the type alias?